linux/step01_centos8_pg_deps.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+PGVER=${PGVER:-pg11}
+
+
+#start-postgresql-installation-general
+#required to install postgresql, rpm Db corrupted
+rm -f /var/lib/rpm/__db*
```

Comment:
I'm concerned about having this in the final docs. Do we know that this won't match any other (non-PG) directories?

Reply:
This is the main RPM db. It's possible to rebuild it if it becomes corrupted: https://serverfault.com/questions/94367/centos-rpm-database-trashed-rpm-rebuilddb-wont-fix-can-i-recover-using-v

But if our instructions lead to that corruption (i.e. it's not already corrupted in the base docker/VM image and it's not a yum/dnf bug) I don't think we should publish them.

Reply:
I moved it one step down to help during the test so it appears in the doc during that phase
i will revert that commit

Reply:
the command ``rm ...`` should no longer appear in the doc now